config/i18n-tasks.yml

```diff
@@ -371,6 +371,8 @@ ignore_unused:
   - decidim.profiles.show.badges
   - decidim.profiles.show.conversations
   - decidim.profiles.show.groups
+  - decidim.profiles.show.group_admins
+  - decidim.profiles.show.group_members
   - decidim.profiles.show.view_full_profile
```

Comment:
I prefer it as you usually do of adding them as a comment in the controller using the `i18n-tasks-use` directive. That way is easy to detect if this is changing and adapt the i18n-task configuration

https://github.com/decidim/decidim/blob/develop/decidim-core/app/controllers/decidim/profiles_controller.rb

Can you move all the `decidim.profiles.show.*` keys 🙏🏽 ?

Reply:
I moved only the ones that were needed in [6b0a8de](https://github.com/decidim/decidim/pull/10120/commits/6b0a8de2c26e9e0f26bd3ea3f2467c9b26569a56)

We need to cleanup the list from i18n-tasks